extension/intents/self/self.js

```diff
@@ -11,7 +11,22 @@ intentRunner.registerIntent({
 intentRunner.registerIntent({
   name: "self.openLexicon",
   async run(context) {
-    await browserUtil.openOrActivateTab("/views/lexicon.html");
+    context.keepPopup()
+    const helloImage = '../../assets/images/hello-card.jpg'
+    const helloCard = {
+      answer: {
+        imgSrc: `${helloImage}`,
+        helloText: "Is it me you're looking for?",
```

Comment:
Could you just use `eduText` or `context.displayText()` for this message?

Reply:
Okay, will do. Just thought helloText seemed appropriate for the context. Thanks. Will make the changes ASAP